crates/ruma-client-api/src/future.rs

```diff
@@ -0,0 +1,38 @@
+//! Endpoints for sending and receiving futures
+
+pub mod future_message_event;
+pub mod future_state_event;
```

Comment:
Endpoint names usually contain a verb, so it could be:

```suggestion
pub mod send_future_message_event;
pub mod send_future_state_event;
```